public/images/warning-sign.svg

```diff
@@ -1,4 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" fill="none" viewBox="0 0 14 14">
-    <path stroke="#FD704A" stroke-width=".98439" d="M7 .492A6.506 6.506 0 0 1 13.508 7 6.507 6.507 0 1 1 7 .492z"/>
-    <path fill="#FD704A" d="M5.706 10.084a1.288 1.288 0 0 1 2.576 0 1.288 1.288 0 1 1-2.576 0zm.405-5.76l-.063-1.712h1.89l-.063 1.712-.263 3.629H6.37l-.259-3.63z"/>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 450 450" width="20" height="20">
+    <path d="M225.24,24.86c-110.46,0-200,89.55-200,200s89.54,200,200,200,200-89.55,200-200S335.69,24.86,225.24,24.86Z" fill="#ed6d47"></path><rect fill="none"></rect><path d="M188.2,313c0-21.92,15.7-36.84,36.8-36.84s36.8,14.92,36.8,36.84-15.7,37.47-36.8,37.47S188.2,335,188.2,313Zm11.56-164.61L198,99.49H252l-1.8,48.94-7.5,103.67H207.26Z" fill="#c4cbd2"></path>
```

Comment:
> fill="#c4cbd2"

This results in a gray `!` over an orange background. If we update to 

> fill="#fff"

We will have a white `!`, which I think looks much better. Comparison below

![Screenshot from 2019-09-23 12-58-20](https://user-images.githubusercontent.com/6762864/65423972-4d518600-de02-11e9-9b1d-797412fc4300.png)

Reply:
@jholdstock Thanks for the feedback, regarding icon coloring it looks much better with white `!`, updating it.